processors/nitf21GeospatialConverters/geographicCoordsConverter.ts

```diff
@@ -61,12 +56,12 @@ export class GeographicCoordsConverter extends BaseProcessor {
                 }
 
                 // Make call to Coordinate Conversion service to get results back in WGS84 Decimal Degrees
-                const coordinateConversionService = new CoordinateConversionService(this.executionContext.correlationId);
+                const coordinateConversionService = new CoordinateConversionService(this.executionContext.correlationId, this.logger);
                 const body = await coordinateConversionService.get(coordinateConversionRequest);
 
                 if (body && body.Coordinates) {
 
-                    const nitf21Helper = new Nitf21ConverterHelper(this.executionContext, this.processorDef);
+                    const nitf21Helper = new Nitf21ConverterHelper(this.executionContext, this.processorDef, this.logger);
```

Comment:
Should this be changed to const nitf21Helper: Nitf21ConverterHelper like the one in Decimal Degrees?  (Same for all other converters.)

Reply:
Type is redundant. Will remove it.